lib/propolis/src/enlightenment/hyperv/mod.rs

```diff
@@ -183,6 +189,12 @@ pub struct HyperV {
 
 impl HyperV {
     /// Creates a new Hyper-V enlightenment stack with the supplied `features`.
+    ///
+    /// The caller must call [`attach`] to finish initializing this
+    /// enlightenment stack before starting any VM components that depend on it.
+    /// Otherwise the stack may panic while the VM is running.
```

Comment:
mmmmmaybe this ought to have a big leading "WARNING:" on it? not a big deal though.